### PR TITLE
MKS-45 Add endpoints

### PR DESCRIPTION
**Pull request by Rostik18**

### CODE REVIEW

File: Rostik_Baitsar/Festival/Festival/Properties/launchSettings.json [hunk 1]
```diff
@@ -12,7 +12,7 @@
     "IIS Express": {
       "commandName": "IISExpress",
       "launchBrowser": true,
-      "launchUrl": "api/Bands",
+      "launchUrl": "api/values",
```
Comment (bvzayats) [inline Rostik_Baitsar/Festival/Festival/Properties/launchSettings.json:15]: Why you launch BandsController in the start. Better launch the swagger.

File: Rostik_Baitsar/Festival/Festival/Options/SwaggerOptions.cs [hunk 2]
```diff
@@ -0,0 +1,12 @@
+﻿
```
Comment (bvzayats) [inline Rostik_Baitsar/Festival/Festival/Options/SwaggerOptions.cs:1]: I think it is another Task feature.

File: Rostik_Baitsar/Festival/Festival/Startup.cs [hunk 3]
```diff
@@ -21,7 +21,14 @@ public Startup(IConfiguration configuration)
         // This method gets called by the runtime. Use this method to add services to the container.
         public void ConfigureServices(IServiceCollection services)
         {
-            services.AddMvc().SetCompatibilityVersion(CompatibilityVersion.Version_2_2);           
+            string connection = Configuration.GetConnectionString("DefaultConnection");
+            services.AddDbContext<BandContext>(options => options.UseSqlServer(connection));
+            services.AddMvc().SetCompatibilityVersion(CompatibilityVersion.Version_2_2);
```
Comment (bvzayats) [inline Rostik_Baitsar/Festival/Festival/Startup.cs:26]: I think it is another Task feature.

File: Rostik_Baitsar/Festival/Festival/Startup.cs [hunk 4]
```diff
@@ -21,7 +21,14 @@ public Startup(IConfiguration configuration)
...
+
+            services.AddSwaggerGen(x =>
```
Comment (bvzayats) [inline Rostik_Baitsar/Festival/Festival/Startup.cs:28]: I think it is another Task feature.

File: Rostik_Baitsar/Festival/Festival/Startup.cs [hunk 5]
```diff
@@ -35,7 +42,20 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env)
             {
                 // The default HSTS value is 30 days. You may want to change this for production scenarios, see https://aka.ms/aspnetcore-hsts.
                 app.UseHsts();
-            }           
+            }
+
+            var swaggerOptions = new Options.SwaggerOptions();
```
Comment (bvzayats) [inline Rostik_Baitsar/Festival/Festival/Startup.cs:47]: I think it is another Task feature.

File: Rostik_Baitsar/Festival/Festival/Options/SwaggerOptions.cs (hunk 2 above)
Comment (Rostik18) [reply Rostik_Baitsar/Festival/Festival/Options/SwaggerOptions.cs:1]: I forgot to add it in the previous step.

**Pull request closed without merging**